pyproject.toml

```diff
@@ -24,9 +24,7 @@ packages = [
 ]
 include = [
   # all is an object -> prevent parse issue with dependabot
-  { path="README.md", format =["sdist","wheel"] },
-  { path="LICENSE", format=["sdist","wheel"] },
```

Comment:
license and notice is no longer needed, since https://github.com/python-poetry/poetry/issues/8441 is done via poetry1.8.1